doc/whats_new/v0.23.rst

```diff
@@ -424,6 +424,17 @@ Changelog
   distance matrix is not square and `affinity=precomputed`.
   :pr:`16257` by :user:`Simona Maggio <simonamaggio>`.
 
+:mod:`sklearn.svm`
+....................
+
+- |Enhancement| invoke scipy blas api for svm kernel function in ``fit``,
+  ``predict``, ``predict_proba``, ``decision_function``, ``cross_validation``,
+  ``libsvm_sparse_train``, ``libsvm_sparse_predict``, 
+  ``libsvm_sparse_predict_proba`` and ``libsvm_sparse_decision_function``
```

Comment:
The `libsvm_*` are not part of the public API so that only leaves `predict`, `predict_proba`, `decision_function` methods I think.

Also wouldn't `nuSVR` and `nuSVC` (and possibly `OneClassSVM`) also be affected?

Reply:
I've checked the code in ```_classes.py``` and ```svm.cpp```. ```nuSVR```, ```nuSVC``` and ```OneClassSVM``` are also affected. Besides, I think ```fit``` is also a public api, which I would like you to confirm.

Reply:
Great. Yes sure fit as well.